ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/FieldTemplate.js

```diff
@@ -1,12 +1,11 @@
 import React from "react";
 
 export const CustomFieldTemplate=(props) => {
-  const {  classNames, help,  description,  children } = props;
+  const {  classNames, children } = props;
   return (
     <div className={classNames}>
-      {description}
+      <div style={{ marginTop : "5px" }}> {" "}  </div>
```

Comment:
Added a margin_top of 5px so that the fields layout is clean and more compact.
It is by_default provided by description, Since I removed the description, so I added a margin Top.

Reply:
Can you use some other more resilient css positioning, like flex?

Reply:
Ideally, rigidity is avoided in this way.

Reply:
Yeah, I have removed it .